packager/packager.js

```diff
@@ -34,6 +34,10 @@ var parseCommandLine = require('./parseCommandLine.js');
 var webSocketProxy = require('./webSocketProxy.js');
 
 var options = parseCommandLine([{
+  command: 'origin',
+  default: 'http://localhost',
+  type: 'string',
+}, {
```

Comment:
@ide What I am hearing you say is that we should model everything after HTML location and replace `origin` with `hostname` and `protocol`. Is that what you are proposing? If so: I like it!